core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -76,7 +76,7 @@ angular.module('oppia').component('signupPage', {
         ctrl.updateWarningText(username);
         if (!ctrl.warningI18nCode) {
           $http.post('usernamehandler/data', {
-            username: ctrl.username
+            username: username
```

Comment:
Why this change?

Reply:
On line 71, if `ctrl.username` is true then this block won't be executed, so it'll be undefined in the HTTP post. I've replaced it by just username because it makes more sense.

Reply:
@kevintab95 could this possibly be related to a server error we were seeing  (https://github.com/oppia/oppia/issues/6757)?